baseplate/lib/prometheus_metrics.py

```diff
@@ -88,3 +89,106 @@ def get_requests_total_metric(self) -> Counter:
     def get_active_requests_metric(self) -> Gauge:
         """Return the active_requests metrics"""
         return thrift_server_active_requests
+
+
+# http server labels and metrics
+http_server_histogram_labels = [
+    "http_method",
+    "http_endpoint",
+    "http_success",
+]
+http_server_requests_total_labels = [
+    "http_method",
+    "http_endpoint",
+    "http_success",
+    "http_response_code",
+]
+http_server_active_requests_labels = [
+    "http_method",
+    "http_endpoint",
+]
+
+http_server_latency_seconds = Histogram(
+    "http_server_latency_seconds",
+    "Time spent processing requests",
+    http_server_histogram_labels,
+    buckets=default_buckets,
+)
+http_server_request_size_bytes = Histogram(
+    "http_server_request_size_bytes",
+    "Size of incoming requests in bytes",
+    http_server_histogram_labels,
+    buckets=default_buckets,
+)
+http_server_response_size_bytes = Histogram(
+    "http_server_response_size_bytes",
+    "Size of outgoing responses in bytes",
+    http_server_histogram_labels,
+    buckets=default_buckets,
+)
```

Comment:
Mentioned in #663 but these buckets aren't suitable for measuring request/response size in bytes

Reply:
Indeed, they're not being called at the moment. Taking them out entirely and will put them back in a separate PR.